src/fsharp/TastOps.fs

```diff
@@ -2504,16 +2504,17 @@ let fullNameOfEntityRef nmF xref =
     | Some pathText -> pathText +.+ nmF xref
 
 let tagEntityRefName (xref: EntityRef) name =
+    let rng = Some ( box xref.DefinitionRange )
```

Comment:
ah, because it's a hack. extra info passed around is just `obj option` because I initially wasn't sure what's needed to navigate.